src/stories/Quiz/Component/Live Example.mdx

```diff
@@ -0,0 +1,10 @@
+import { Meta, Controls, Canvas } from '@storybook/blocks';
+import * as ComponentStories from './ComponentStories.stories';
+
+<Meta of={ComponentStories} />
+
+### Interactive Live Example
```

Comment:
Can we fix the height?
![quiz](https://github.com/Constructor-io/constructorio-ui-quizzes/assets/2954126/6bd3aafd-2e66-45d3-a74f-0db231bb9916)